src/styles/main.scss

```diff
@@ -1,3 +1,215 @@
-body {
+$pixel-proportion: 50px;
+$font-weight-proportion: 100;
+$shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+$card-width: 4 * $pixel-proportion;
+$card-container-padding-right: $pixel-proportion * 0.96;
+$card-height: $pixel-proportion * 8.16;
+$main-min-width: $card-width * 4 + $card-container-padding-right * 3;
+$main-padding-left: calc((100vw - #{$main-min-width}) / 2);
+$blue-color: #00acdc;
+$black-color: #060b35;
+$almost-white-color: #f3f3f3;
+$white-color: #fff;
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 $pixel-proportion;
+  align-items: center;
+  box-shadow: $shadow;
+}
+
+.logo {
+  height: $pixel-proportion * 0.8;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: $pixel-proportion * 0.24;
+  font-weight: $font-weight-proportion * 5;
+  line-height: $pixel-proportion * 1.2;
+  color: $black-color;
+}
+
+.is-active {
+  position: relative;
+  color: $blue-color;
+
+  &::after {
+    position: absolute;
+    content: "";
+    background-color: $blue-color;
+    height: $pixel-proportion * 0.08;
+    width: 100%;
+    border-radius: $pixel-proportion * 0.08;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.nav__item:not(:last-child) {
+  margin-right: $pixel-proportion * 0.4;
+}
+
+.nav__link:hover {
+  color: $blue-color;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: $main-min-width;
+  padding-top: $pixel-proportion;
+  padding-bottom: $pixel-proportion * 1.46;
+  padding-right: $main-padding-left;
+  padding-left: $main-padding-left;
```

Comment:
what's going on here? why do you need other paddings except for padding-top?
now your page differs from the design
remove all padding except the top one and add max-width to the main container
![image](https://user-images.githubusercontent.com/57598825/148753103-dc75d575-4ba1-41d7-9661-ea43d5178f72.png)